task11.js

```diff
@@ -0,0 +1,13 @@
+// Write a function canDrive(age, hasLicense) that returns true if the person is 18 or older and has a valid license, otherwise false.
+
+const canDrive = (age, hasLicense) => {
+  if (typeof age !== "number" || Number.isNaN(age) || !Number.isInteger(age)) {
+    throw new Error(
+      "Provided value must be a valid integer, that starts from 1"
+    );
+  }
+
+  return age >= 18 && hasLicense ? true : false;
```

Comment:
`hasLicense  `already evaluates to `true `or `false`, the ternary is unnecessary
just `return age >= 18 && hasLicense;`